src/neo-vm/Types/StackItem.cs

```diff
@@ -21,6 +23,16 @@ public virtual StackItem ConvertTo(StackItemType type)
             throw new InvalidCastException();
         }
 
+        public StackItem DeepCopy()
```

Comment:
`Clone`?

Reply:
`Clone` and `Struct.Clone()` conflict, and their functions are different.